sunpy/map/mapsequence.py

```diff
@@ -359,21 +360,18 @@ def annotate_frame(i):
         def updatefig(i, im, annotate, ani_data, removes):
             while removes:
                 removes.pop(0).remove()
-
             im.set_array(ani_data[i].data)
-            im.set_cmap(kwargs.get('cmap', ani_data[i].plot_settings['cmap']))
-            norm = deepcopy(kwargs.get('norm', ani_data[i].plot_settings['norm']))
-
+            im.set_cmap(kwargs.get('cmap', ani_data[i].plot_settings.get('cmap'))) or "grey"
```

Comment:
Not sure if this is a good idea but maybe this could should not error if it's missing a colormap?